peachjam/templates/peachjam/court_class_detail.html

```diff
@@ -6,3 +6,7 @@
 {% block month_list %}
   {% include "peachjam/_court_class_month_list.html" %}
 {% endblock %}
+{% block follow-button %}
```

Comment:
I think an include would be better than a block (or a block that wraps an include)